layouts/community/ergodox/adnw_p_u_q/keymap.c

```diff
@@ -0,0 +1,371 @@
+#include QMK_KEYBOARD_H
+#include "debug.h"
+#include "action_layer.h"
+#include "keymap_german.h"
+
+// Note:
+// You need to set a Tapping-Value:
+// TAPPING_TERM-Variable in keyboards/ergodox_infinity/config.h 
+// My ideal value is 145 but find your own
+
+// more options here: https://docs.qmk.fm/config_options.html
+#define FORCE_NKRO
```

Comment:
These defines should be in the `config.h` file in this folder, otherwise they won't get picked up properly